pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: pool
-version: 1.3.1
+version: 1.3.3
 author: Dart Team <misc@dartlang.org>
 description: A class for managing a finite pool of resources.
 homepage: https://github.com/dart-lang/pool
 dependencies:
   async: "^1.4.0"
   stack_trace: ">=0.9.2 <2.0.0"
 environment:
-  sdk: ">=1.22.0 <2.0.0"
+  sdk: ">=2.0.0-dev.0.1 <2.0.0-dev.infinity"
```

Comment:
2 things

1) This won't be a . release. It'll be dev.1.0 – I think
2) Don't need the infinity thing any more. Just <2.0.0 will work...

Reply:
...also, update changelog!